src/services/playlist.js

```diff
@@ -0,0 +1,99 @@
+import axios from "axios";
+import { playlistUrl } from "../utils/index";
+import { playlistConstants } from "../utils/dataActions";
+
+const {
+  GET_PLAYLISTS,
+  ADD_PLAYLIST,
+  DELETE_PLAYLIST,
+  GET_PLAYLIST_BY_ID,
+  ADD_VIDEO_TO_PLAYLIST,
+  DELETE_VIDEO_FROM_PLAYLIST,
+} = playlistConstants;
+
+const config = {
+  headers: { authorization: localStorage.getItem("token") },
```

Comment:
suggestion: you can make a function `getToken` and use it everywhere

Reply:
Function won't be efficient I guess, I would have to call the function inside api calls, that's why used object, but I have to make the object constant and use it everywhere. Thanks for pointing out.